masonry/src/contexts.rs

```diff
@@ -764,59 +767,48 @@ impl_context_method!(
     PaintCtx<'_>,
     AccessCtx<'_>,
     {
+        // TODO - Update once we introduce the is_hovered/has_hovered distinction.
         /// The "hovered" status of a widget.
         ///
-        /// A widget is "hovered" when the mouse is hovered over it. Widgets will
+        /// A widget is "hovered" when a pointer is hovering over it. Widgets will
         /// often change their appearance as a visual indication that they
-        /// will respond to mouse interaction.
+        /// will respond to pointer (usually mouse) interaction.
         ///
         /// The hovered status is computed from the widget's layout rect. In a
         /// container hierarchy, all widgets with layout rects containing the
-        /// mouse position have hovered status.
+        /// pointer position have hovered status.
+        ///
+        /// If the pointer is [captured], then only that widget and its parents
+        /// can have hovered status. If the pointer is captured but not hovering
+        /// over the captured widget, then no widget has the hovered status.
         ///
-        /// Discussion: there is currently some confusion about whether a
-        /// widget can be considered hovered when some other widget has captured the
-        /// pointer (for example, when clicking one widget and dragging to the
-        /// next). The documentation should clearly state the resolution.
+        /// [captured]: crate::doc::doc_06_masonry_concepts#pointer-capture
         pub fn is_hovered(&self) -> bool {
             self.widget_state.is_hovered
         }
 
-        /// Whether the pointer is captured by this widget.
+        /// Whether the mouse is [captured] by this widget.
```

Comment:
Is there a reason the other changes switch from mouse to pointer, and this line changes from pointer to mouse?